packages/typespec-vscode/src/utils.ts

```diff
@@ -389,3 +408,36 @@ export async function loadPackageJsonFile(
   if (!packageJson) return undefined;
   return packageJson as NodePackage;
 }
+
+export async function parseOpenApi3File(filePath: string): Promise<string | undefined> {
```

Comment:
Please also make it more general, like parseJsonFromFile(filePath:string)

Reply:
renamed to suggested name